data-processing/pulsar-flink/2.7.0/pulsar-flink.md

```diff
@@ -200,6 +200,49 @@ In some scenarios, users need messages to be strictly guaranteed message order t
 
 Pulsar Flink connector supports this feature as well. This feature can be enabled by configuring the `enable-key-hash-range=true` parameter. When enabled, the range of Key Hash processed by each consumer is divided based on the parallelism of the task.
 
+## Fault tolerance
+
+Pulsar Flink connector 2.7.0 provides different semantics for source and sink.
+
+### Source
+
+For Pulsar source, Pulsar Flink connector 2.7.0 provides `exactly-once` semantic.
```

Comment:
do we need more info about how to implement `exactly-once` semantic in Pulsar source?

Reply:
@Huanli-Meng thanks for your suggestion, there is no semantic configuration on the source side.